pandas/_libs/parsers.pyx

```diff
@@ -1215,6 +1215,18 @@ cdef class TextReader:
                 cats, codes, dtype, true_values=true_values)
             return cat, na_count
 
+        elif is_extension_array_dtype(dtype):
+            result, na_count = self._string_convert(i, start, end, na_filter,
+                                                    na_hashset)
+            try:
+                # use _from_sequence_of_strings if the class defines it
+                result = dtype.construct_array_type() \
+                              ._from_sequence_of_strings(result, dtype=dtype)
+            except NotImplementedError:
+                result = dtype.construct_array_type() \
+                              ._from_sequence(result, dtype=dtype)
```

Comment:
Rather than falling back, we could also raise an informative error message? (or simply let the NotImplementedError through to the user, assuming this already has a nice error message)